app/javascript/articles/Article.jsx

```diff
@@ -122,6 +122,7 @@ export const Article = ({
         isBookmarked={isBookmarked}
         onClick={bookmarkClick}
       />
+      {/* add button goes here */}
```

Comment:
This should be removed since we decided to go a different direction with the `save` button.